HISTORY.md

```diff
@@ -3,6 +3,7 @@
 
   * Add new upstream MIME types
   * Add `application/toml` with extension `.toml`
+  * Add `image/dds` with extension `dds`
```

Comment:
Since 1.41.1 has already been released, you don't want to list this changed under that version. You can just leave this out.

Reply:
I will remove, thank you.